src/main/java/io/github/fileanalysissuite/adaptersdk/interfaces/framework/OptionsProvider.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.github.fileanalysissuite.adaptersdk.interfaces.framework;
+
+import java.util.Optional;
+
+public interface OptionsProvider {
+
+    /**
+     * Retrieves available option names.
+     * @return the option names.
+     */
+    Iterable<String> getOptionNames();
+
+    /**
+     * Retrieves a value of an option.
+     * <p>
+     * If the option does not exist, <c>null</c> will be returned.
```

Comment:
The build is still failing on my machine because of this:
`OptionsProvider.java:31: error: unknown tag: c`